var/spack/repos/builtin/packages/qrupdate/package.py

```diff
@@ -29,17 +29,31 @@ def edit(self, spec, prefix):
         return
 
     def install(self, spec, prefix):
+
         lapack_blas = spec['lapack'].libs + spec['blas'].libs
-        # Build static and dynamic libraries
-        make('lib', 'solib',
-             'BLAS={0}'.format(lapack_blas.ld_flags),
-             'LAPACK={0}'.format(lapack_blas.ld_flags))
-        # "INSTALL" confuses "make install" on case-insensitive filesystems
+
+        make_args = [
+            'BLAS={0}'.format(lapack_blas.ld_flags),
+            'LAPACK={0}'.format(lapack_blas.ld_flags)
+        ]
+
+        # If 64-bit BLAS is used:
+        if (spec.satisfies('^openblas+ilp64') or
+            spec.satisfies('^intel-mkl+ilp64') or
+            spec.satisfies('^intel-parallel-studio+mkl+ilp64')):
+            make_args.append('FFLAGS=-fdefault-integer-8')
```

Comment:
Wondering if this should be an option or attribute exposed by `blas` providers. In that way one could write something similar to:
```python
if spec['blas'].ilp64:
    make_args.append('FFLAGS=-fdefault-integer-8')

```

Reply:
@alalazo  This feature would indeed be nice. Should I open a separate issue for it and this patch gets applied as is for now?

Reply:
Fair enough. If you could open an issue and ping me back for merge it would be great!